### PR TITLE
added amsterdam page, change links, add mike

### PR DESCRIPTION
**Pull request by bitschmidty**

### CODE REVIEW

File: 2019_amsterdam.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <h1 class="section-heading">June 2019 Amsterdam</h1>
+                    <br />
+                    <h2>June 5-7 2019</h2>
+                    <p>CoreDev Hacking Meetings <em>(invite only)</em></p>
+                    <hr />
+                    <h4>June 8-9 2019</h4>
+                    <p><a href="https://breaking-bitcoin.com/">Breaking Bitcoin 2</a> (security conference on Bitcoin)
```
Comment (jnewbery) [inline 2019_amsterdam.html]: Not sure how many of these we want to include, since they're not coredev events.

File: 2019_amsterdam.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                    </p>
+                    <hr />
+                    <h4>June 11-13 2019</h4>
+                    <p><a href="https://breaking-bitcoin.com/otherPages/2019/training.html">Bitcoin dev
+                            training & workshop</a> by Justin Moon,
+                        Stephane Roche and Mickael Folkson (similar to the amazing Bitcoin
```
Comment (jnewbery) [inline 2019_amsterdam.html]: Remove everything in the parentheses here.

**Review by jnewbery — COMMENTED** [review]
Looks good. A couple of comments inline.

File: 2019_amsterdam.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    </section>
+    <section id="participants">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+
+                        <h2 class="section-heading">Sponsors</h2>
+                        <hr />
+                        <p><a href="https://building-on-bitcoin.com/"><img src="https://breaking-bitcoin.com/img/Logo_horiz_white_min.png" /></a></p>    
+                    <br /><br />
```
Comment (moneyball) [inline 2019_amsterdam.html:123]: Should this instead be the breaking bitcoin web site? https://breaking-bitcoin.com

File: 2019_amsterdam.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                    <hr />
+                    <h4>June 8-9 2019</h4>
+                    <p><a href="https://breaking-bitcoin.com/">Breaking Bitcoin 2</a> (security conference on Bitcoin)
+                    </p>
+                    <hr />
+                    <h4>June 11-13 2019</h4>
+                    <p><a href="https://breaking-bitcoin.com/otherPages/2019/training.html">Bitcoin dev
+                            training & workshop</a> by Justin Moon,
+                        Stephane Roche and Mickael Folkson (similar to the amazing Bitcoin
+                        Edge, but much more hand-on)</p>
```
Comment (moneyball) [inline 2019_amsterdam.html]: hands-on?

**Comment by moneyball** [comment]
tACK. Couple of minor points of feedback but otherwise ready to merge.

File: 2019_amsterdam.html (hunk 1 above)
Comment (jonasschnelli) [reply 2019_amsterdam.html]: +1 on avoiding non-core-dev events.

File: 2019_amsterdam.html (hunk 4 above)
Comment (bitschmidty) [reply 2019_amsterdam.html]: From our invite email. Removing per John.

File: 2019_amsterdam.html (hunk 3 above)
Comment (bitschmidty) [reply 2019_amsterdam.html:123]: Ill confirm with Kevin.

File: 2019_amsterdam.html (hunk 1 above)
Comment (bitschmidty) [reply 2019_amsterdam.html]: Took out everything except coredev and the coworking space. Let me know if that works.

**Comment by jnewbery** [comment]
tACK 258c5140ab868ffe0b1d98508d8d5d2e2c513536

**Comment by jonasschnelli** [comment]
looks good. ACK.

**Comment by jnewbery** [comment]
Looks good @bitschmidty . I've added an extra commit with some minor edits. I'll merge now since there have only been very minor edits since Steve and Jonas have ACKed.

**Comment by jnewbery** [comment]
@jonasschnelli - please deploy!

**Comment by jonasschnelli** [comment]
Deployed!

**Pull request merged**